tests/acceptance/forward_server.py

```diff
@@ -0,0 +1,409 @@
+"""TCP/IP forwarding/echo service for testing."""
```

Comment:
@gmr, I pulled the code in this module from my own https://github.com/vitaly-krugl/inetpy repo that presently uses an "MIT License". Do I need to put any sort of copyright header in this file? Thanks, Vitaly